### ISSUE TITLE
How to use it?

### ISSUE THREAD

**Issue by pavlyutkin** [gsurma/style_transfer#3]
I wanna use it for generating images with styles

**Comment by quintendewilde** [comment]
Deep respect for your work! 
And also interested in how to run this....

google collab? requirements.txt? wich python version, etc..

I'm using this for non-commeercial ends

**Comment by clamytoe** [comment]
I agree a requirements.txt or environment.yml file is needed. I'm running into some tensorflow errors about using the `+=` operator. I was able to get past them by replacing them with `.assign_add()`. I then got an error about tf.gradient not being supported with eager execution, to use `tf.GradientTape` instead. The `GradientTape` way didn't work, so I disabled eager execution, only to run into a runtime error about attempting to capture an EagerTensor without building a function...

**Comment by Tanapl02** [comment]
Start now

**Comment by Salphoris55** [comment]
> I agree a requirements.txt or environment.yml file is needed. I'm running into some tensorflow errors about using the `+=` operator. I was able to get past them by replacing them with `.assign_add()`. I then got an error about tf.gradient not being supported with eager execution, to use `tf.GradientTape` instead. The `GradientTape` way didn't work, so I disabled eager execution, only to run into a runtime error about attempting to capture an EagerTensor without building a function...

Have this same issue. Haven't figured out how to get past it so far. If I do I will report here.
